Fix broken urls in the examples page and translate the page to pt-BR

nicolasgandrade opened the pull request:
<!--
Thank you for your pull request. Please provide a description and 
note the Certificate of Origin below. 

-->

<!--
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
-->

This PR replaces 2 links in the examples page that were broken with new ones, removes other 2 that used to redirect to examples that **are not** in the Prisma repository anymore, and also translates the `examples.md` page to Portuguese.

I could have changed the links of the examples that are not anymore in the Prisma repo to one pointing an old commit, however I decided not to do so because since they were permanently removed, I assumed that those examples are not supported anymore. Also, there's no new alternative in the Prisma repo.

Related issue: https://github.com/expressjs/expressjs.com/issues/1716

nicolasgandrade committed:
fix: replace broken links with new ones and remove references to deleted examples in examples.md

nicolasgandrade committed:
feat: translate the examples.md page to pt-br

nicolasgandrade committed:
feat: list the examples page in the header-pt-br.html file

bjohansebas reviewed (approved):
LGTM

carlosstenzel commented:
@bjohansebas 

If we were to create a new page 'example ' for pt-br, we would have to create it for all other languages, and adjust the menu as well.

bjohansebas commented:
Sorry for the late response, I had taken a break. It's true that the pages should be added to the other languages in English, but since the author had only focused on those links and adding them to their language, I didn't want to ask them to add it to the other languages. @nicolasgandrade, would you like to add the 'examples' page to the other languages?

carlosstenzel commented:
@bjohansebas 

I am proposing an improvement in another PR, adding only the menu item. https://github.com/expressjs/expressjs.com/pull/1724

(Considering that this PR has no modifications, you could close it and just add the reference )

nicolasgandrade commented:
Hey guys, my bad for the delay, I was in a hurry with a delivery in another project.

The #1724 PR makes sense. Actually, I didn't know we could add the item to the menu referencing another language, but yes, I agree that's better than having the item only in the EN and PT pages.

Since @carlosstenzel has already done those changes in his PR, I think I don't have anything else to add, but if there's something else that I'm missing, ping me here.

carlosstenzel commented:
@nicolasgandrade 

Poderia fazer nesse PR 
- remover a alteração no menu
- atualizar a branch
- Atualizar os links das paginas existentes de exemplo em en, pt-br e outras que tenham o mesmo.
- alterar o nome do PR 

Acho que essas alterações fazem mais sentido para seguirmos com o PR.

Obrigado pela sua contribuição ;)

bjohansebas commented:
Here it is ready, @carlosstenzel, can you confirm that the Portuguese translation is correct so we can merge it?

nicolasgandrade committed:
Merge branch 'gh-pages' into fix/broken-examples-urls

nicolasgandrade committed:
refactor: remove the more-examples item from the header-pt-br

Please, find the #1724 PR, where the items for the page are being added to all supported languages.

nicolasgandrade committed:
fix: update the broken urls with correct ones in the indonesian examples page

nicolasgandrade commented:
@carlosstenzel Right, I removed the change in the menu, updated the branch and also updated the links in the Indonesian page (this one was the only one translated, the other ones are all pointing to the english version, which was updated in this PR).

Just a question: since the ideia (as far as I understood) is to have the menu link pointing to the english page in all languages for now, should I leave the portuguese translation in this PR as well even not being used now or should I just remove the file from this PR?

bjohansebas commented:
Please, keep the Portuguese translation
